lib/langchain.rb

```diff
@@ -7,7 +7,10 @@
 require "json"
 
 loader = Zeitwerk::Loader.for_gem
+
 loader.ignore("#{__dir__}/langchainrb.rb")
+loader.ignore("#{__dir__}/langchain/assistants/llm")
```

Comment:
It would be nice to make it work with zeitwerk without just ignoring it if possible. I'm wondering if that whole folder structure needs to change `/lib/langchain/assistants/`?